codegen/cmd/injection-gen/generators/client.go

```diff
@@ -95,7 +95,7 @@ func Get(ctx context.Context) {{.clientSetInterface|raw}} {
 	untyped := ctx.Value(Key{})
 	if untyped == nil {
 		{{.loggingFromContext|raw}}(ctx).Fatalf(
-			"Unable to fetch %T from context.", ({{.clientSetInterface|raw}})(nil))
+			"Unable to fetch {{.clientSetInterface}} from context.")
```

Comment:
This can now be `.Fatal` (everywhere)

Reply:
true

Reply:
Any thoughts to panic'ing? If tests aren't setup incorrectly then it just exits.